src/status.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line import/no-cycle, import/extensions
+import { input, renderTodos, todos } from './index.js';
+
+function saveTodo() {
+  const todoValue = input.value;
+  todos.push({
+    value: todoValue,
+    completed: false,
+    id: Date.now(),
+  });
+  input.value = '';
+  renderTodos();
+}
+
+export default saveTodo;
```

Comment:
- [ ] Good Job for Adding a JavaScript file that contain methods related to the status updates (completed: true / false). But When I marked the checked box it will not update  items object's value for completed key upon user actions. Kindly work on it because it' s stated in the Project Requirement
Your To-do list before user actions : 
![Capture](https://user-images.githubusercontent.com/93598189/195692351-981cc3f6-5e12-43fd-a316-1d05bdd78621.PNG)

After I try to check the first To-do List (after user action): it will not Update items object's value for completed key
 
![Capture](https://user-images.githubusercontent.com/93598189/195693087-a64a0674-833d-440c-952d-a6ada8381edd.PNG)

- [ ] Please also make sure that you Implemented a function for the "Clear all completed" button (using filter() method) as stated in the Project requirement, Because when I clicked on button it remove all to-do list items(both completed and not completed items) But it must only filter out the completed one and Cleared out , Kindly work on it.

Reply:
ok I fixed everything and the clear button works with the filter method